src/pages/dar_application/DataAccessRequest_new.js

```diff
@@ -0,0 +1,338 @@
+          validators: [FormValidators.REQUIRED],
+          description: 'Please enter below a non-technical summary of your RUS suitable for understanding by the general public (written at a high school reading level or below).',
+          placeholder: 'Please limit your your non-technical summary to 1100 characters',
+          rows: 6,
+          maxLength: 1100,
+          ariaLevel: ariaLevel + 3,
+          defaultValue: formData.nonTechRus,
+          onChange,
+        }),
+
+        h(FormFieldTitle, {
+          title: '2.5 Data Use Acknowledgements',
+          description: 'Please confirm listed acknowledgements and/or document requirements below:',
+          isRendered: needsGsoAcknowledgement() || needsDsAcknowledgement() || needsPubAcknowledgement(),
+        }),
+
+        h(FormField, {
+          id: 'gsoAcknowledgement',
+          type: FormFieldTypes.CHECKBOX,
+          isRendered: needsGsoAcknowledgement(),
+          toggleText: 'I acknowledge that I have selected a dataset limited to use on genetic studies only (GSO). I attest that I will respect this data use condition.',
+          defaultValue: formData.gsoAcknowledgement,
+          onChange,
+        }),
+
+        h(FormField, {
+          id: 'pubAcknowledgement',
+          isRendered: needsPubAcknowledgement(),
+          type: FormFieldTypes.CHECKBOX,
+          toggleText: 'I acknowledge that I have selected a dataset which requires results of studies using the data to be made available to the larger scientific community (PUB). I attest that I will respect this data use condition.',
+          defaultValue: formData.pubAcknowledgement,
+          onChange,
+        }),
+
+        h(FormField, {
+          id: 'dsAckowledgement',
+          isRendered: needsDsAcknowledgement(),
+          type: FormFieldTypes.CHECKBOX,
+          toggleText: 'I acknowledge that the dataset can only be used in research consistent with the Data Use Limitations (DULs) and cannot be combined with other datasets of other phenotypes. Research uses inconsistent with DUL are considered a violation of the Data Use Certification agreement and any additional terms descried in the addendum',
+          defaultValue: formData.dsAckowledgement,
+          onChange,
+        }),
+
+
+        h(FormFieldTitle, {
+          description: 'One or more of the datasets you selected requires local IRB approval for use. Please upload your local IRB approval(s) here as a single document. When IRB approval is required and Expedited of Full Review is required, it must be completed annually. Determinations of Not Human Subjects Research (NHSR) by IRBs will not be accepted as IRB approval.',
+          isRendered: needsIrbApprovalDocument(),
+        }),
+        div({
+          isRendered: needsIrbApprovalDocument(),
+          style: {
+            display: 'flex',
+            justifyContent: 'space-between',
+          }
+        }, [
+          div({}, [
+            h(FormField, {
```

Comment:
The FormField.FILE components need to have `defaultValues` as well if the user has already uploaded a file to their draft DAR.

Reply:
should have default values now!